module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -495,7 +497,9 @@ namespace module::input {
                          *                  Kinematics                  *
                          ************************************************/
 
+                        // Htx is a Map from ServoID to (a homogeneous transform from that joint to the torso)
```

Comment:
What?

Reply:
I was having trouble understanding what it was, and once I found out I added this comment, but I agree that my wording is awkward, although I think it's accurate. Do you have a better explanation of `Htx`? I struggled to get better words for it without being _that_ literal

Reply:
"Htx is a map from ServoID to homogeneous transforms from each ServoID to the torso", or something like that?